src/app/core/components/login/login.component.ts

```diff
@@ -13,11 +14,19 @@ export class LoginComponent implements OnInit {
     password: ''
   };
 
-  constructor(private auth: AuthService) {}
+  constructor(private auth: AuthService, private router: Router) { }
 
-  ngOnInit() {}
+  ngOnInit() { }
 
   googleLogin() {
     this.auth.googleLogin().catch(error => (this.loginError = error));
```

Comment:
Add a navigation similar to emailLogin here too

Reply:
Fixed